torch/lib/THCUNN/generic/SpatialGridSamplerBilinear.cu

```diff
@@ -0,0 +1,93 @@
+#ifndef THC_GENERIC_FILE
+#define THC_GENERIC_FILE "generic/SpatialGridSamplerBilinear.cu"
+#else
+
+static inline void THNN_(SpatialGridSamplerBilinear_shapeCheck)(
+    THCState *state,
+    THCTensor *input,
+    THCTensor *grid,
+    THCTensor *gradOutput) {
+  THCUNN_argCheck(state, THCTensor_(nDimension)(state, input) == 4, 2, input,
+      "4D input tensor expected but got: %s");
+  THCUNN_argCheck(state, THCTensor_(nDimension)(state, grid) == 4, 2, grid,
+      "4D grid tensor expected but got: %s");
+
+  int nbatch   = THCTensor_(size)(state, input, 0);
```

Comment:
Same here about `int64_t`. There was a recent refactoring that replaced `long` by a fixed type.